test/CacheFactoryTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * container-interop-doctrine
+ *
+ * @link      http://github.com/DASPRiD/container-interop-doctrine For the canonical source repository
+ * @copyright 2016 Ben Scholzen 'DASPRiD'
+ * @license   http://opensource.org/licenses/BSD-2-Clause Simplified BSD License
+ */
+
+namespace ContainerInteropDoctrineTest;
+
+use ContainerInteropDoctrine\AbstractFactory;
+use ContainerInteropDoctrine\CacheFactory;
+use Doctrine\Common\Cache\FilesystemCache;
+use Interop\Container\ContainerInterface;
+use PHPUnit_Framework_TestCase;
+
+class CacheFactoryTest extends PHPUnit_Framework_TestCase
```

Comment:
Not sure how @DASPRiD feels about it, but I'd suggest using `@covers` annotation at the class level to avoid coverage leakage